inc/Ui/Index.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace dokuwiki\Ui;
+
+use dokuwiki\Extension\Event;
+use dokuwiki\Form\Form;
+
+/**
+ * DokuWiki Index Insterface
+ *
+ * @package dokuwiki\Ui
+ */
+class Index extends Ui
+{
+    /**
+     * Display page index
+     *
+     * @author   Andreas Gohr <andi@splitbrain.org>
+     *
+     * @param string $ns
+     * @return void
+     */
+        public function show($ns = '')
```

Comment:
Minor: the indentation is unregular

Reply:
Fixed in https://github.com/splitbrain/dokuwiki/pull/3198/commits/9e3166413adffd9d067b4cde8da8d04c44d9c8bf